PluginCore/PluginCore/Managers/ShortcutKeysManager.cs

```diff
@@ -189,6 +211,28 @@ public static bool IsValidExtendedShortcutSecond(Keys second)
             return true;
         }
 
+        /// <summary>
+        /// Retrieves a value indicating whether the specified virtual key code and keyboard state has a corresponding Unicode character or characters.
+        /// </summary>
+        /// <param name="keyData">The virtual key code to be translated.</param>
+        public static bool IsCharacterKeys(Keys keyData)
+        {
+            byte[] keyStates = new byte[256];
+            if ((keyData & Keys.Shift) == Keys.Shift)
+            {
+                keyStates[(int) Keys.ShiftKey] = 0xFF;
+            }
+            if ((keyData & Keys.Control) == Keys.Control)
+            {
+                keyStates[(int) Keys.ControlKey] = 0xFF;
+            }
+            if ((keyData & Keys.Alt) == Keys.Alt)
+            {
+                keyStates[(int) Keys.Menu] = 0xFF;
+            }

```

Comment:
I didn't feel the need to call [GetKeyboardState()](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646299(v=vs.85).aspx) as I could just do it this way. 
Would there be any problem doing this?